### PR TITLE
fixing multiple jobs in a task

### PR DESCRIPTION
**Pull request by denizea**
This fix helps leek to run multiple jobs in a task

### CODE REVIEW

File: django_leek/server.py [hunk 1]
```diff
@@ -21,8 +22,9 @@ def target(queue):
     log.info('Worker Starts')
     done = False
     while not done:
-        task_id = queue.get()
-        if task_id is None:
+        try:
+            task_id = queue.get(block=True,timeout=1)
```
Comment (oskli) [inline django_leek/server.py:26]: Missing space between arguments.

File: django_leek/server.py [hunk 2]
```diff
@@ -59,10 +62,10 @@ class Pool(object):
     def __init__(self):
         if platform == 'darwin':
             # OSX does not support forking
-            self.queue = queue.Queue()
+            self.queue = queue.Queue(maxsize=10000)
```
Comment (oskli) [inline django_leek/server.py:65]: Is there a motivation to selecting 10000? I suggest that this is defined as a variable.

File: django_leek/server.py [hunk 3]
```diff
@@ -105,13 +108,12 @@ def handle(self):
...
             self.request.send(json.dumps(response).encode())
 
         except OSError as e:
             # in case of network error, just log
             log.exception("network error")
 
     def finish(self):
-        for pool in self.pools.values():
-            pool.queue.put(None)
+        None
```
Comment (oskli) [inline django_leek/server.py:119]: This looks a bit wierd. This function is already defined in super, not doing anything. So if this is what we want I think this whole function could be removed?

File: django_leek/server.py (hunk 2 above)
Comment (denizea) [reply django_leek/server.py:65]: For large batch requests we might need high number of queue. Let me name it as a defined variable.

File: django_leek/server.py (hunk 3 above)
Comment (EvelineV) [reply django_leek/server.py:120]: or `pass` would look better than `None`. But yeah, removing probably makes sense

**Pull request closed without merging**